main.js

```diff
@@ -1 +1,91 @@
+/* Objectives of this assignment:
 
+ -  understand how to use JS functions
+*/
+function functionName () {
+    console.log("");
+} 
+functionName
+
+// understand how to use JS algorithms
+
+for (function functionName "");
+ 
+ /* add:
+    condition
+    parameter
+    argument
+
+    Note: a statement goes here
+*/
+
+
+//  functions have to be called or invoked
+functionName ("" , "");
+
+// function output can be displayed to user and internnally to developer
+console.log ("");
+
+// or
+return ("")
+
+// * practice working with the DOM - the Document Object Model
+// * practice responding to user input
+
+
+// Code for a rectangle.
+/*  In small group on 9/16/20, we thought we needed to code a border around the calculator because that is how the .jpg appeared when viewed in VSCode but when you open the .jpg in Windows Viewer you see there is no border around the calculator */
+
+
+
+
+// Make the rectangle green inside
+
+
+
+// Put a grid inside the rectangle for the keys and output
+
+
+
+// Make the grid clickable
+
+
+// Somehow/somewhere there is a loop
+// while loop -- increments to increase to a certain number
+let i = 0;
+while (i < 10){
+    console.log(i);
+    i++;
+}
+
+// Somehow/somewhere this a declared and called constant variable
+/*  -- Note per Rebecca - var is out of date (var was recommended by Mark Myers JS book; I like his book and will use it (I like his writing style and focus on syntax in coding.). I will compare his suggestions to MDN to get the up to date version of his instruction) */
+
+/*  Getting an error message on "5" below. 
+    Question to Rebecca/Mari - is "" equivalent to ' '
+    SubAnswer:  I see my typo of "4," that created the error message (I still have question above)
+
+    Question: is ',' a 'thing' (method, expression, parameter in JS?)
+*/
+const someWhere = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"];
+
+for (let pickOne of someWhere) {
+    console.log(pickOne)
+}
+
+// somewhere when a user 'clicks' then output on the calculator happens
+// and the console.log of a 'click' is part of the calculator output
+// So ... a click & eventListener is required.
+
+/* does all of the code below go in the body of my html (index.html) file? */
+// kp - yes; or maybe no
+
+let image = document.querySelector ("img")
+<- undefined  /* I don't know if the arrow should be coded on is generated by VS Code */
```

Comment:
What image is this referring to? This same strategy is how you would get the buttons.